diff --git a/src/declarativeimports/plasmacomponents3/ScrollBar.qml b/src/declarativeimports/plasmacomponents3/ScrollBar.qml
--- a/src/declarativeimports/plasmacomponents3/ScrollBar.qml
+++ b/src/declarativeimports/plasmacomponents3/ScrollBar.qml
@@ -20,15 +20,15 @@
 import QtQuick 2.6
 import QtQuick.Templates 2.0 as T
 import org.kde.plasma.core 2.0 as PlasmaCore
-
+import org.kde.kirigami 2.2 as Kirigami
 
 T.ScrollBar {
     id: control
 
     implicitWidth: background.implicitWidth
     implicitHeight: background.implicitHeight
 
-    hoverEnabled: true
+    hoverEnabled: !Kirigami.Settings.isMobile
 
     visible: control.size < 1.0
 
@@ -40,10 +40,10 @@
         visible: control.size < 1.0
 
         prefix: control.horizontal ? "background-horizontal" : "background-vertical"
-        opacity: control.hovered ? 1 : 0
+        opacity: Kirigami.Settings.isMobile ? (control.active ? 1 : 0) : (control.hovered ? 1 : 0)
         Behavior on opacity {
             OpacityAnimator {
-                duration: 250
+                duration: units.longDuration
             }
         }
     }
@@ -56,6 +56,12 @@
         visible: control.size < 1.0
 
         prefix: control.hovered ? "mouseover-slider" : "slider"
+        opacity: Kirigami.Settings.isMobile ? (control.active ? 1 : 0) : 1
+        Behavior on opacity {
+            OpacityAnimator {
+                duration: units.longDuration
+            }
+        }
     }
 
     PlasmaCore.Svg {
diff --git a/src/declarativeimports/plasmacomponents3/private/DefaultListItemBackground.qml b/src/declarativeimports/plasmacomponents3/private/DefaultListItemBackground.qml
--- a/src/declarativeimports/plasmacomponents3/private/DefaultListItemBackground.qml
+++ b/src/declarativeimports/plasmacomponents3/private/DefaultListItemBackground.qml
@@ -22,6 +22,7 @@
 import QtQuick.Controls 1.0 as Controls
 import QtQuick.Controls.Private 1.0
 import org.kde.plasma.core 2.0 as PlasmaCore
+import org.kde.kirigami 2.2 as Kirigami
 
 PlasmaCore.FrameSvgItem {
     id: background
@@ -34,7 +35,7 @@
 
     PlasmaCore.FrameSvgItem {
         imagePath: "widgets/listitem"
-        visible: !Settings.isMobile
+        visible: !Kirigami.Settings.isMobile
         prefix: "hover"
         anchors.fill: parent
         opacity: control.hovered && !control.pressed ? 1 : 0

