commit ab302872d300158a590af841bb890392d11489be
Author: David Rosca <nowrep@gmail.com>
Date:   Sun Mar 6 16:13:54 2016 +0100

    Fix iconitem test - second try

diff --git a/autotests/data/icons/test-theme/index.theme b/autotests/data/icons/test-theme/index.theme
index 52a7275..5eaeff3 100644
--- a/autotests/data/icons/test-theme/index.theme
+++ b/autotests/data/icons/test-theme/index.theme
@@ -3,8 +3,6 @@ Name=Unittest Theme
 
 DisplayDepth=32
 
-Inherits=breeze
-
 DesktopDefault=48
 DesktopSizes=16,22,32,48,64,128,256
 ToolbarDefault=22
@@ -21,7 +19,7 @@ DialogSizes=16,22,32,48,64,128,256
 ########## Directories
 ########## ordered by category and alphabetically
 
-Directories=apps/22,apps/32
+Directories=apps/22,apps/32,apps/48
 
 [apps/22]
 Size=22
diff --git a/autotests/iconitemtest.cpp b/autotests/iconitemtest.cpp
index 5853f74..ae825a9 100644
--- a/autotests/iconitemtest.cpp
+++ b/autotests/iconitemtest.cpp
@@ -61,6 +61,9 @@ void IconItemTest::initTestCase()
 
     QFile::remove(configPath);
 
+    QIcon::setThemeSearchPaths({QFINDTESTDATA("data/icons")});
+    QIcon::setThemeName("test-theme");
+
     KConfigGroup cg(KSharedConfig::openConfig(), "Icons");
     cg.writeEntry("Theme", "test-theme");
     KConfigGroup plasmaConfig(KSharedConfig::openConfig("plasmarc"), "Theme");
